### PR TITLE
PXC-2948: wsrep_min_log_verbosity - add variable

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

**Review by kennt-percona — APPROVED** [review]
lgtm

File: doc/source/wsrep-system-index.rst [hunk 1]
```diff
@@ -375,6 +375,57 @@ Possible values for this variable are:
...
 
+.. variable:: wsrep_min_log_verbosity
+
+   :cli: ``--wsrep-min-log-verbosity``
+   :conf: Yes
+   :scope: Global
+   :dyn: Yes
+   :default: 3
+
+This variable defines the *minimum* logging verbosity for the given
```
Comment (kamil-holubicki) [inline doc/source/wsrep-system-index.rst]: Not exactly. This variable affects only logs that come from wsrep/galera replication part. Server logs are untouched.
Please see the description in the last comment of https://jira.percona.com/browse/PXC-2948
'wsrep_min_log_verbosity' defines the minimum logging verbosity from wsrep/galera. Its default value is 3

File: doc/source/wsrep-system-index.rst [hunk 2]
```diff
@@ -375,6 +375,57 @@ Possible values for this variable are:
...
+|pxc| server and has the same values as the ``log_error_verbosity``
+variable.
+
+The effective log verbosity cannot be lower than the value of
```
Comment (kamil-holubicki) [inline doc/source/wsrep-system-index.rst]: As above. New variable affects only wsrep/galera part.

It defines minimum desired verbosity from wsrep/galera regardless of what is set to log_error_verbosity'.

Example:
wsrep_min_log_verbosity = 3;
log_error_verbosity = 1;
We see only error/system logs from server parts + all logs from wsrep/galera part

wsrep_min_log_verbosity = 1;
log_error_verbosity = 3;
We see all logs from server part and all logs from wsrep/galera part

Please refer to the table included in the Jira ticket.

**Review by kamil-holubicki — CHANGES_REQUESTED** [review]

**Review by kamil-holubicki — APPROVED** [review]
lgtm

**Pull request merged**